### PR TITLE
Build apache integration

### PR DESCRIPTION
**Pull request by mtojek**
This PR migrates apache modules to the integration.

Screenshots:

Zrzut ekranu 2020-06-24 o 12 22 11

Zrzut ekranu 2020-06-23 o 12 24 45

screencapture-localhost-5601-app-dashboards-2020-06-24-12_00_50

screencapture-localhost-5601-app-dashboards-2020-06-24-12_20_08

### CODE REVIEW

**Comment by mtojek** [comment]
Currently blocked by: https://github.com/elastic/kibana/issues/69677

**Comment by mtojek** [comment]
The integration is finished. All dashboards are presenting data.

@ruflin few questions here:
1. Under which version should I release the package?
2. I marked it as `release: beta`, so once I merge this PR it will be released to public. Is it ok?

**Comment by ruflin** [comment]
* Version: As it is still in beta, I would pick something 0.*.0. I see the others are in 0.1.0 so this is probably a good pick
* Beta: Yes, we want to have this in 7.9.

File: packages/nginx/manifest.yml [hunk 1]
```diff
@@ -46,6 +46,6 @@ datasources:
       required: true
       show_user: true
       default:
-      - http://127.0.0.1
+      - http://127.0.0.1:80
```
Comment (exekias) [inline packages/nginx/manifest.yml:49]: is this change intended?

Comment (mtojek) [reply packages/nginx/manifest.yml:49]: Intended to slightly suggest that it's possible to configure a port. I can clean this one if it's better.
